### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by baldjeff**
Checking my box

### CODE REVIEW

File: 2020_Fall/readme.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Kimball, Ellen
 
-- [ ] Liew, Jeffrey
+- [x] Liew, Jeffrey
```
Comment (joewadcan) [inline 2020_Fall/readme.md]: @baldjeff Looks like you've missed adding the GitHub username and emoji. Just make another commit on this branch.

**Comment by rahulvenk** [comment]
+1
